src/app/general/pages/gen-profile/gen-profile.component.ts

```diff
@@ -1,143 +1,155 @@
 //Core Imports
 import {
-  Component,
-  OnInit
+    Component,
+    OnInit
 } from '@angular/core';
 
 import {
-  NgModel
+    NgModel
 } from '@angular/forms';
 
 //Application Imports
 import {
-  Course,
-  Quest,
-  User,
-  TOTXP,
-  MAXXP
+    Course,
+    Quest,
+    User,
+    TOTXP,
+    MAXXP

```

Comment:
opinion based comment as to how this totxp and maxxp should be implemented or structured